openedx_learning/apps/authoring/components/api.py

```diff
@@ -191,6 +192,20 @@ def create_next_component_version(
             # content represented by our key from the next version. Otherwise,
             # we add our key->content_pk mapping to the next version.
             if content_pk is not None:
+                if isinstance(content_pk, bytes):
```

Comment:
[request]: Please rename `content_pk` to something that indicates that it could be bytes or a primary key, e.g. `content_pk_or_bytes`.

Then later on, you can do:

```python
if isinstance(content_pk_or_bytes, bytes):
    ....
    content_pk = content.pk
else:
    content_pk = content_pk_or_bytes
```

... or something like that.